### DISCUSSION TITLE
Website for Lark

### DISCUSSION THREAD

**Discussion by erezsh** [lark-parser/lark#1027] [General]
We are now the proud owners of https://www.lark-parser.org/

(Sorry, lark.org is taken!)

What should we put there? We can do the obvious landing page with slides and a few links, like every other project these days.

But, I'm open to suggestions. All thoughts / ideas / remarks are welcome.

**Comment by ThatXliner** [comment]
Woah... how'd you host it? I've always wanted to get the `CNAME` working but can never manage

**Reply by erezsh** [reply]
It was a bit of a headache! But only because it's completely arbitrary and doesn't make sense. But it's not that hard.

These are the DNS records I used -


A | lark-parser.org| directs to 185.199.108.153
CNAME | www.lark-parser.org | is an alias of lark-parser.github.io
